### DISCUSSION TITLE
Making @font-face work on github pages

### DISCUSSION THREAD

**Discussion by ashkenazzio** [vercel/next.js#43913] [Help]
Hello,
I have a small app which uses a local `@font-face`.  Everything works fine on dev environment of course.
I've deployed my app using the github actions next.js workflow and the css and images are loaded fine but the font is not.

The font .woff file is stored in `/public/webfonts/fonts`.
The  `@font-face` import is in the `globals.css` file.

Any tweaks I have to do to make the font load properly?

**Comment by douglasramaldes** [comment]
Move fonts to the src directory and use relative paths. This ensures Next.js properly processes and bundles the fonts.

create the assets folder and put the font there
```src/assets/fonts``` 

In my case, I change 
```  src: url('/fonts/brown-sugar.ttf') format('truetype'); ```
to 
```  src: url('../assets/fonts/your-font.ttf') format('truetype'); ```
